### PR TITLE
feat: added nvmrc and prepublish script

### PR DESCRIPTION
**Pull request by bradleySuira**
### Description

- [x] Add .nvmrc file and added webpack config and scripts to eslintignore
- [x] Add prePublishOnly script

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ coverage
 examples
 doc
 .eslintrc.js
+scripts/updateERC20.js
+webpack.config.js
```
Comment (harshjv) [inline .eslintignore:8]: please add a new line after this last line

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v16.13.0
```
Comment (harshjv) [inline .nvmrc:1]: please add a new line after this last line

**Review by harshjv — CHANGES_REQUESTED** [review]

File: .eslintignore (hunk 1 above)
Comment (bradleySuira) [reply .eslintignore]: done

File: .nvmrc (hunk 2 above)
Comment (bradleySuira) [reply .nvmrc]: done

**Review by harshjv — APPROVED** [review]
LGTM

**Pull request merged**